code/modules/vehicles/bike.dm

```diff
@@ -79,6 +79,10 @@
 	if(user == load)
 		unload(load)
 		user << "You unbuckle yourself from \the [src]"
+	else if(user != load && load)
+		unload(load)
```

Comment:
Because space bikes are a combat tool, do you think you can make it so that unbuckling someone from them takes a bit of time? Just enough to give the other side a chance to get the fuck away, if they're conscious. Look into `do_after()`, I think was the proc.